engines/python/src/main/java/ai/djl/python/engine/Connection.java

```diff
@@ -160,7 +161,7 @@ static String[] getPythonStartCmd(
                 }
                 sb.append(host).append(':').append(localSize);
             }
-            String[] args = new String[50];
+            String[] args = new String[51];
             args[0] = "mpirun";
             args[1] = "-np";
             args[2] = String.valueOf(worldSize);
```

Comment:
I wonder if we can also make this a list instead of hard coding the indices from 0 to 51..

Reply:
yes, I think we should take this up in a follow up PR. I'm trying to keep refactoring in this PR to a minimum to avoid non-essential changes from distracting.